packages/caver-klay/caver-klay-kct/contract/token/KIP17/IERC721Receiver.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title ERC721 token receiver interface
+ * @dev Interface for any contract that wants to support safeTransfers
+ * from ERC721 asset contracts.
+ */
+contract IERC721Receiver {
```

Comment:
Don't we have to have KIP17TokenReceiver?
https://github.com/klaytn/kips/pull/18/files#diff-e34690f57510c63897293c98402f37e8R163

Reply:
Oh, sorry we have it. Please ignore this. :)